README.md

```diff
@@ -35,6 +35,8 @@ Some general notes and limitations of event webhooks:
 
 This event is triggered after a booking has been made within the Healthengine platform. An example payload is shown below.
 
+Please note that the patient gender field is optional. By default it will only be provided if your booking form collects it and if it is required as part of the services you offer.
```

Comment:
I don't think this is required other fields are optional currently with no note.

Reply:
Hmm that's a fair point

Reply:
```suggestion
```